cddl/contrib/opensolaris/tools/ctf/cvt/ctf.c

```diff
@@ -1316,16 +1316,20 @@ resurrect_types(ctf_header_t *h, tdata_t *td, tdesc_t **tdarr, int tdsize,
 			tdp->t_fndef = xcalloc(sizeof (fndef_t));
 			tdp->t_fndef->fn_ret = tdarr[type];
 
-			v = (void *) (dptr + (idwidth * (vlen - 1)));
-			if (vlen > 0 && *(uint_t *)v == 0)
-				tdp->t_fndef->fn_vargs = 1;
+			if (vlen > 0) {
+				argid = 0;
```

Comment:
This is because of CTFv2? Should be a Fixes for https://github.com/CTSRD-CHERI/cheribsd/commit/bdf290cd3e1a69d41c2f8bb60bd415cfa78adba2 and upstreamed.

Reply:
I saw that the size is being checked below, but I'm not sure if it's also needed for this check (I would guess so but I'm not sure). CC: @markjdb